x-pack/solutions/search/plugins/search_getting_started/kibana.jsonc

```diff
@@ -8,19 +8,9 @@
     "id": "searchGettingStarted",
     "server": true,
     "browser": true,
-    "configPath": [
-      "xpack",
-      "searchGettingStarted"
-    ],
-    "requiredPlugins": [],
-    "optionalPlugins": [
-      "cloud",
-      "console",
-      "searchNavigation",
-      "usageCollection"
-    ],
-    "requiredBundles": [
-      "kibanaReact"
-    ]
+    "configPath": ["xpack", "searchGettingStarted"],
```

Comment:
You'll probably want to revert this formatting change. I'm guessing it was done accidentally as it happened on one of my PRs before. I think single-line formatting is the preference in Kibana.